src/main/kotlin/com/csbroker/apiserver/repository/common/RedisRepository.kt

```diff
@@ -32,4 +34,23 @@ class RedisRepository(
     fun removePasswordVerification(code: String) {
         redisTemplate.delete(code)
     }
+
+    fun setRank(scoreMap: Map<UUID, Double>) {
+        scoreMap.forEach {
+            redisTemplate.opsForZSet().add("ranking", it.key.toString(), it.value)
+        }
+    }
+
+    fun getRank(userId: UUID): RankResultDto {
+        var rank: Long? = null
+
+        val score = redisTemplate.opsForZSet().score("ranking", userId.toString()) ?: 0.0
+        val rankKey = redisTemplate.opsForZSet().reverseRangeByScore("ranking", score, score, 0, 1)?.first()
+
+        if (rankKey != null) {
+            rank = redisTemplate.opsForZSet().reverseRank("ranking", rankKey)?.plus(1)
```

Comment:
rankKey가 null이 아닌건 redis에서 키가 나왔다는 것이니 null이 아님을 주장하는게 좋지 않나요??
```suggestion
            rank = redisTemplate.opsForZSet().reverseRank("ranking", rankKey)!!.plus(1)
```
코틀린 문법을 잘 몰라서 맞다면 적용 시켜주시고 아니면 리젝해주셔도 됩니다!

Reply:
@kshired 맞는지 틀린지 알려줘여..

Reply:
!! 으로 단언하는거도 괜찮은 것 같긴한데, 그렇게 해버리면 혹시나 모르는 문제에서 npe가 발생할 수도 있어서.. 그냥 null로 return하는게 서버단에서는 안전할것 같았습니다!